android/app/src/main/java/org/fossasia/openevent/core/auth/profile/EditProfileActivity.java

```diff
@@ -88,6 +91,28 @@ protected void onCreate(Bundle savedInstanceState) {
             Views.hideKeyboard(this, this.getCurrentFocus());
             saveChanges();
         });
+
+
+        ActivityCompat.requestPermissions(EditProfileActivity.this,
+                new String[]{Manifest.permission.READ_EXTERNAL_STORAGE},1);
```

Comment:
What is 1?

Reply:
Its actually the request code of the permission